install_config/configuring_sdn.adoc

```diff
@@ -346,14 +351,17 @@ iptables -t nat -A POSTROUTING -o eth1 -j MASQUERADE
 ----
 
 To persist those changes in the *_/etc/sysconfig/iptables_* use the following
-snippet on every node:
+command on every node:
 
 ----
 cp /etc/sysconfig/iptables{,.orig}
 sh -c "tac /etc/sysconfig/iptables.orig | sed -e '0,/:DOCKER -/ s/:DOCKER -/:DOCKER ACCEPT/' | awk '"\!"p && /POSTROUTING/{print \"-A POSTROUTING -o eth1 -j MASQUERADE\"; p=1} 1' | tac > /etc/sysconfig/iptables"
 ----
 
-NOTE: iptables-save saves all the current "in memory" iptables rules, but, as
-both Docker, Kubernetes and {product-title} creates a lot of iptables rules
-(services, etc.) not designed to be persisted, you will end up with a messy
-/etc/sysconfig/iptables file.
+[NOTE]
+====
+The `iptables-save` command saves all the current _in memory_ iptables rules.
+However, because Docker, Kubernetes and {product-title} create a high number of iptables rules
+(services, etc.) not designed to be persisted, saving these rules can result in a large
```

Comment:
I'd say in a "bad" file instead large, because the problem is if you persist the k8s services, docker, etc. rules, you will be screwed 🗡